diff.html

```diff
@@ -344,15 +344,33 @@ <h5>Warframe Drop Data - Compare Versions</h5>
                     else if (a.relicName) { return a.relicName.localeCompare(b.relicName) }
                 })
             }
+
             for (let sub of Object.keys(build[key])) {
                 // Sort the subitems.
                 if (Array.isArray(build[key][sub].mods)) { newArray[key][sub].mods = build[key][sub].mods.sort(sortModName) }
                 else if (Array.isArray(build[key][sub].enemies)) { newArray[key][sub].enemies = build[key][sub].enemies.sort(sortEnemyName) }
                 else if (Array.isArray(build[key][sub].items)) { newArray[key][sub].items = build[key][sub].items.sort(sortItemName) }
                 else if (Array.isArray(build[key][sub].rewards)) { newArray[key][sub].rewards = build[key][sub].rewards.sort(sortItemName) }
-                else if (build[key][sub]["rewards"] && Array.isArray(build[key][sub]["rewards"].A)) { newArray[key][sub]["rewards"].A = build[key][sub]["rewards"].A.sort(sortItemName) }
-                else if (build[key][sub]["rewards"] && Array.isArray(build[key][sub]["rewards"].B)) { newArray[key][sub]["rewards"].B = build[key][sub]["rewards"].B.sort(sortItemName) }
-                else if (build[key][sub]["rewards"] && Array.isArray(build[key][sub]["rewards"].C)) { newArray[key][sub]["rewards"].C = build[key][sub]["rewards"].C.sort(sortItemName) }
+                else if (build[key][sub]["rewards"] && Array.isArray(build[key][sub]["rewards"]["A"]) && Array.isArray(build[key][sub]["rewards"]["B"]) && Array.isArray(build[key][sub]["rewards"]["C"])) {
+                    newArray[key][sub]["rewards"]["A"] = build[key][sub]["rewards"]["A"].sort(sortItemName)
+                    newArray[key][sub]["rewards"]["B"] = build[key][sub]["rewards"]["B"].sort(sortItemName)
+                    newArray[key][sub]["rewards"]["C"] = build[key][sub]["rewards"]["C"].sort(sortItemName)
+                }
+                else {
+                    if (key === "missionRewards") {
```

Comment:
this should be combined with the above line to make it
```js
} else if (key === "missionRewards") {
```

Reply:
Good observation. However, I'm still getting some data in there that's not been handed, so I'm still not completely done with that construction. As soon as I've handled the last remaining data, I will convert it for consistency.

Reply:
All done. Changed it for consistency.